config/config.go

```diff
@@ -56,9 +56,6 @@ type Config struct {
 	// MetricsConfig specifies configuration for metrics emission
 	MetricsConfig PrometheusConfig `json:"metricsConfig,omitempty"`
 
-	// FeatureFlags for general/experimental features
-	FeatureFlags FeatureFlags `json:"featureFlags,omitempty"`
```

Comment:
Are we getting rid of feature flags entirely? Wouldn't we want to keep this for future features?

Reply:
I don't think it is bad design. You can use normal config item or a envvar? Both are easier and more flexible.

Reply:
Are we enabling resource duration by default?